test/cets_test_helper.erl

```diff
@@ -0,0 +1,37 @@
+-module(cets_test_helper).
+-include_lib("eunit/include/eunit.hrl").
+
+-export([
+    get_disco_timestamp/3,
+    assert_unique/1
+]).
+
+-export([
+    set_nodedown_timestamp/3,
+    set_other_servers/2,
+    set_join_ref/2
+]).
+
+get_disco_timestamp(Disco, MapName, NodeKey) ->
+    Info = cets_discovery:system_info(Disco),
+    #{MapName := #{NodeKey := Timestamp}} = Info,
+    Timestamp.
+
+%% Fails if List has duplicates
+assert_unique(List) ->
+    ?assertEqual([], List -- lists:usort(List)),
+    List.
+
+%% Overwrites nodedown timestamp for the Node in the discovery server state
+set_nodedown_timestamp(Disco, Node, NewTimestamp) ->
```

Comment:
This seems to be used in one place in disco_SUITE. I'd rather move it there, but I'm also fine with keeping the suites clean from helpers if we prefer this style.